bin/generate-gutenboarding-design-thumbnails.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+/**
+ * Generate static design screenshot thumbnails for Gutenboarding
+ * Until we can generate tall screenshots on the fly with mshots:
+ * https://github.com/Automattic/mShots/issues/16
+ * https://github.com/Automattic/wp-calypso/issues/40564
+ *
+ * You'll need Puppeteer installed.
+ * Change `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD`in `package.json` to `false` and run `npm ci`
+ */
+
+const designs = require( '../client/landing/gutenboarding/available-designs.json' );
+const puppeteer = require( 'puppeteer' );
+const wpUrl = require( '@wordpress/url' );
+
+const screenshotsPath = './static/page-templates/design-screenshots';
+
+const getDesignUrl = design => {
+	return wpUrl.addQueryArgs( design.src, {
+		font_headings: design.fonts[ 0 ],
+		font_base: design.fonts[ 1 ],
+	} );
+};
+
+async function run() {
+	console.log( 'Taking screenshots...' );
+	const browser = await puppeteer.launch( { headless: true } );
+
+	await Promise.all(
+		designs.featured.map( async design => {
+			const url = getDesignUrl( design );
+			const screenshotFile = `${ screenshotsPath }/${ design.slug }_${ design.template }_${ design.theme }.jpg`;
+
+			console.log( `Opening ${ url }` );
+			const page = await browser.newPage();
+			await page.setViewport( {
+				width: 980,
+				height: 2200,
```

Comment:
Might actually need to set viewport height to something smaller, take the screenshot with `fullPage: false` and then crop it before saving.

Reply:
BTW, I would use [`capture-screen`](https://github.com/sindresorhus/capture-website). Would save you from a lot of headache.

Reply:
Ah, nice one! I agree but now this script is already done and the viewport settings seem to be just the ones passed to Puppeteer, it won't do cropping for us anyway.

Reply:
@alshakero thanks for the tip — ended up using that after all.